### PR TITLE
create development namespace for learning from cloud platform guide

### PR DESCRIPTION
**Pull request by wood-n**

### CODE REVIEW

**Comment by wood-n** [comment]
Thank you for your message, please can I request a second review? Thanks

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/hello-world-test/00-namespace.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: hello-world-test
+  labels:
+    cloud-platform.justice.gov.uk/is-production: "false"
+    cloud-platform.justice.gov.uk/environment-name: "development"
+  annotations:
+    cloud-platform.justice.gov.uk/business-unit: "HMPPS Digital"
+    cloud-platform.justice.gov.uk/slack-channel: "${slack-channel}"
```
Comment (digitalronin) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/hello-world-test/00-namespace.yaml:10]: Please replace `${slack-channel}` with the name of a team slack channel we can use to get in touch if there are any problems with this namespace, e.g. `#hmpps_dev`

Comment (wood-n) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/hello-world-test/00-namespace.yaml]: Hi, I have updated and added a private channel. Please just let me know if the team channel needs to be public?

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/hello-world-test/00-namespace.yaml [hunk 2]
```diff
@@ -7,7 +7,7 @@ metadata:
     cloud-platform.justice.gov.uk/environment-name: "development"
   annotations:
     cloud-platform.justice.gov.uk/business-unit: "HMPPS Digital"
-    cloud-platform.justice.gov.uk/slack-channel: "${slack-channel}"
+    cloud-platform.justice.gov.uk/slack-channel: "hello-world-test"
```
Comment (digitalronin) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/hello-world-test/00-namespace.yaml:10]: This needs to be an actual slack channel that exists on the mojdt slack instance. e.g. `#hmpps_dev` would be a good choice.

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/hello-world-test/resources/ecr.tf [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+data:
+  repo: <ECR_REPO_BASE64_ENCODED>
+  token: <SLACK_TOKEN_BASE64_ENDCODED>
+Note that the <ECR_REPO_BASE64_ENCODED> and <SLACK_TOKEN_BASE64_ENDCODED> must be encoded as base64.
+e.g 'echo -n <SLACK_TOKEN> | base64'
+As this file will contain the slack token it is important that it is encyrpted within the repo that has git-encrypt. Also the file must reside within your own team's repo and not a repo that is shared between teams such as the 'cloud-platform-environments'.
+Save the above secret yaml with the desired name and create the secret as follows: 
+kubectl create -f <SLACK_SECRET_FILE_NAME>
+Lastly, after you have created your kubernetes slack secret as above, move the following lambda module outside the comments section so that it is created alongside your ECR resource. 
+module "ecr_scan_lambda" {
```
Comment (digitalronin) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/hello-world-test/resources/ecr.tf:62]: The closing `*/` is missing here, which is why the plan pipeline shows an error:
https://concourse.cloud-platform.service.justice.gov.uk/builds/7335

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/hello-world-test/resources/ecr.tf [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+  providers = {
+    aws = aws.london
+  }
+}
+
+resource "kubernetes_secret" "example_team_ecr_credentials" {
+  metadata {
+    name      = "example-team-ecr-credentials-output"
+    namespace = "<NAMESPACE>"
```
Comment (poornima-krishnasamy) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/hello-world-test/resources/ecr.tf:27]: Please update your actual namespace name here. "hello-world-test"

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/hello-world-test/resources/ecr.tf (hunk 3 above)
Comment (wood-n) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/hello-world-test/resources/ecr.tf]: Apologies, this file was committed in error

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/hello-world-test/resources/ecr.tf (hunk 4 above)
Comment (wood-n) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/hello-world-test/resources/ecr.tf]: Apologies, this file was committed in error

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/hello-world-test/00-namespace.yaml (hunk 2 above)
Comment (wood-n) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/hello-world-test/00-namespace.yaml]: I created a private channel in slack for this purpose, as this is purely for running through the hello world project on the Cloud Platform wiki. Is a private channel unsuitable?

**Comment by astroash** [comment]
Can we get a rereview on this PR? It would be good to be able to continue on Monday

**Comment by astroash** [comment]
Is this PR ready to be merged?

**Pull request merged**